### PR TITLE
ED-2069: Add OCI oss support in Deploy/DataPipeline/FlinkPipelineJobs

### PR DESCRIPTION
**Pull request by ddevadat**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: kubernetes/helm_charts/secor/config/secor.common.properties [hunk 1]
```diff
@@ -357,7 +357,8 @@ secor.max.message.size.bytes=100000
 
 # Class that will manage uploads. Default is to use the hadoop
 # interface to S3.
-secor.upload.manager.class=com.pinterest.secor.uploader.AzureUploadManager
+# secor.upload.manager.class=com.pinterest.secor.uploader.AzureUploadManager
```
Comment (anandp504) [inline kubernetes/helm_charts/secor/config/secor.common.properties:360]: @ddevadat Don't we have to use CSP specific upload manager? It looks like you have changed it from AzureUploadManager to S3UploadManager.

File: kubernetes/helm_charts/secor/config/secor.common.properties [hunk 2]
```diff
@@ -23,12 +23,12 @@ secor.kafka.topic_blacklist=
...
 # Choose what to fill according to the service you are using
 # in the choice option you can fill S3, GS, Swift or Azure
-cloud.service=Azure
+cloud.service={{ $.Values.storage_type }}
 
 # AWS authentication credentials.
 # Leave empty if using IAM role-based authentication with s3a filesystem.
-aws.access.key=
-aws.secret.key=
+aws.access.key={{ $.Values.s3_access_key }}
```
Comment (anandp504) [inline kubernetes/helm_charts/secor/config/secor.common.properties]: @ddevadat  The secor.common.properties should be left as-is and we should add secor.s3.properties to accommodate s3-like object stores. Please refer to secor.azure.properties file and also secor.partition.properties file where secor.azure.properties is imported.

File: kubernetes/helm_charts/secor/config/secor.partition.properties [hunk 3]
```diff
@@ -23,7 +25,7 @@ secor.kafka.group={{ get (get $.Values.secor_jobs $.Release.Name) "consumer_grou
 secor.message.parser.class={{ get (get $.Values.secor_jobs $.Release.Name) "message_parser" }}
 
 # S3 path where sequence files are stored.
-secor.s3.path=
+secor.s3.path={{- get (get $.Values.secor_jobs $.Release.Name) "base_path" }}
```
Comment (anandp504) [inline kubernetes/helm_charts/secor/config/secor.partition.properties]: @ddevadat secor.s3.path also needs to be moved to secor.s3.properties file and configured in the s3 related config file.

**Review by anandp504 — CHANGES_REQUESTED** [review]
@ddevadat Please resolve the PR comments.

File: kubernetes/helm_charts/secor/config/secor.common.properties (hunk 1 above)
Comment (ddevadat) [reply kubernetes/helm_charts/secor/config/secor.common.properties:360]: secor doesnt have an oci specific upload manager. OCI object storage is s3 api compliant. This means OCI can use S3UploadManager

File: kubernetes/helm_charts/secor/config/secor.common.properties (hunk 2 above)
Comment (ddevadat) [reply kubernetes/helm_charts/secor/config/secor.common.properties]: its restructured now

File: kubernetes/helm_charts/secor/config/secor.partition.properties (hunk 3 above)
Comment (ddevadat) [reply kubernetes/helm_charts/secor/config/secor.partition.properties]: its restructured now

File: ansible/roles/portal-dashboard/tasks/main.yml
Comment (ddevadat) [inline ansible/roles/portal-dashboard/tasks/main.yml:1]: This change is done to accomodate PR https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/pull/1615

 Deploy/DataPipeline/AdhocScripts fails on ubuntu 22.04

Ubuntu 22.04 is coming with default python 3.10 version and virtual environment creation is always failing when explicitly virtualenv_python value to pip task in ansible. As per the documentation of ansible(https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pip_module.html),
virtualenv_python : The Python executable used for creating the virtual environment. For example python3.5, python2.7. When not specified, the Python version used to run the ansible module is used.

**Pull request merged**